### ISSUE TITLE
Test use of a formal signature in emails to authors

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#778]
Big institutions tend to use big email signatures with phone numbers, addresses & privacy / legal signs offs in their emails. We have a short sign off, that is just polite but probably detracts for our legitimacy. 

What might a longer sign off look like here? 

e.g:

> Kind Regard, 
> Natalia, 
> Open Access Button Request System Manager
> 
> T <myphonenumber>
> E: Natalia@openaccessbutton.org
> 
> The Open Access Button is supported by SPARC. SPARC is a global coalition of Libraries committed to making Open the default for research and education. SPARC empowers people to solve big problems and make new discoveries through the adoption of policies and practices that advance Open Access, Open Data, and Open Education.
> 
> SPARC is a NGO with ##### based at <address>
> 
> Learn more about the Open Access Button, a tool connecting people to the research articles and data they need. You're receiving this email because an Open Access Button user requested access to research you are an author of. If you'd like to stop receiving emails from Open Access Button, you can let us know.

**Comment by natalianorori** [comment]
I agree with the long signature, perhaps we could add the button's twitter handle as well? @JosephMcArthur

**Comment by JosephMcArthur** [comment]
+1 on twitter, high follower count for sure shows legitimacy.

**Comment by JosephMcArthur** [comment]
This is easy to test & implement while tackling an issue I think is probably still quite significant, I'm currently considering it relatively high priority test.

**Comment by chealsye** [comment]
What do you need from me?

**Comment by JosephMcArthur** [comment]
thoughts, if ya'got'em :) perhaps some ideas for copy & structuring the signature too.

**Comment by chealsye** [comment]
The email signature is a fine idea. It is long, but it is different being a signature. I wouldn't write the signature copy this way, but if this is how you want to explain the Button and SPARC that's fine.

**Comment by natalianorori** [comment]
I would like to test this next week if possible, thoughts on this? @JosephMcArthur?  Would like to have a template up on the system to start testing.

**Comment by natalianorori** [comment]
Joe and I decided that this would be a great candidate for our next test, I'd like to deliver it by the end of this week/beggining of next week.  Any extra thoughts on the final copy for the signature? @chealsye @JosephMcArthur

**Comment by JosephMcArthur** [comment]
- [ ] give nick a heads up about this test - me

**Comment by natalianorori** [comment]
@JosephMcArthur have you talked to Nick about this? This is an easy/small side test to run while we wait for results for bigger ones

**Comment by JosephMcArthur** [comment]
Have requested sign off on doing this
